HarfangLab/harfanglab/_meta/smart-descriptions.json

```diff
@@ -26,6 +26,25 @@
       }
     ]
   },
+  {
+    "value": "{user.target.name} failed to log on {host.hostname} (LogonType {action.properties.LogonType})",
+    "relationships": [
+      {
+        "source": "user.target.name",
+        "target": "host.hostname",
+        "type": "failed to log on to"
+      }
+    ],
+    "conditions": [
+      {
+        "field": "event.provider"
+      },
+      {
+        "field": "event.code",
+        "value": "4625"
```

Comment:
`user.target.name`, `host.hostname` and `action.properties.LogonType` are missing from the conditions.